elizhahn/static-comp
# Feature/collective cards

## Is this a fix or a feature?
- feature

## What is the change?
- Cards were built in HTML and styled with CSS to match the provided comp
- Each card features an image with an arrow icon to visit their collection, and how many views it has gotten
- artist name
- artist location
- artist personal link
- and total amount of artwork in their collection that is displayed online

## What does it fix?

## Where should the reviewer start?
- index.html lines: 37-289
- styles.css lines: 486- 540

## How should this be tested?
- open up index.html on Chrome, Safari, and Firefox, the display should all be the same with no break in layout or awkward elements
- Resize each window to see that each browser is also responsive for tablets and phones

## Screenshots (if appropriate):
